static/app/components/onboarding/productSelection.tsx

```diff
@@ -19,11 +19,13 @@ export enum ProductSolution {
   ERROR_MONITORING = 'error-monitoring',
   PERFORMANCE_MONITORING = 'performance-monitoring',
   SESSION_REPLAY = 'session-replay',
+  PROFILING = 'profiling',
 }
 
 // This is the list of products that are available for each platform
-// Since the ProductSelection component is rendered in the onboarding flow, it is ok to have this list here
+// Since the ProductSelection component is rendered in the onboarding/project creation flow only, it is ok to have this list here
 export const platformProductAvailability = {
```

Comment:
nit: should we keep this alphabetically sorted? we usually do for a good overview and easy edits